src/ttf.cpp

```diff
@@ -84,11 +92,14 @@ TtfFont *TtfFontList::LoadFont(const std::string &font)
     LoadAll();
 
     TtfFont *tf = std::find_if(l.begin(), l.end(),
-        [&](const TtfFont &tf) { return tf.FontFileBaseName() == font; });
+        [&font](const TtfFont &tf) { return tf.FontFileBaseName() == font; });
 
     if(tf != l.end()) {
         if(tf->fontFace == NULL) {
-            tf->LoadFromFile(fontLibrary, /*nameOnly=*/false);
+            if(tf->fontFile.raw.compare(0, 6, "res://"))
```

Comment:
Here, it seems to me like all logic handling the `res:` prefix ought to be concentrated in the same place. (And if not, at least `LoadFromResource` should use an assertion before chopping off the prefix.)

Reply:
That seems reasonable.